docs/evaluation/simulate.rst

```diff
@@ -106,39 +165,57 @@ slightly different answers. With default settings, the input model is
 convolved with the RMF and multiplied by the ARF, and properly scaled
 for the exposure time.
 
-  >>> from sherpa.astro.fake import fake_pha
-  >>> fake_pha(data, srcmdl)
+.. plot::
+    :include-source:
+    :context:
+    :nofigs:
+
+    >>> from sherpa.astro.fake import fake_pha
+    >>> fake_pha(data, full_model)
 
 We bin the counts into bins of at least 5 counts per bin and display
 an image of the simulated spectrum (see :ref:`model_evaluate_example_pha`
-for details)::
-
-  >>> data.set_analysis('energy')
-  >>> data.notice(0.3, 8)
-  >>> data.group_counts(5)
-  >>> from sherpa.plot import DataPlot
-  >>> dplot = DataPlot()
-  >>> dplot.prepare(data)
-  >>> dplot.plot(xlog=True, ylog=True)
+for details):
+
+.. plot::
+    :include-source:
+    :context:
+    :nofigs:
+
+    >>> data.set_analysis('energy')
+    >>> data.notice(0.3, 8)
+    >>> data.group_counts(5)
+    >>> from sherpa.plot import DataPlot
+    >>> nobgplot = DataPlot()
+    >>> nobgplot.prepare(data)
+
+We can overplot the model (both grouped and ungrouped):
+
+.. plot::
+   :context:
+   :include-source:
+
+   >>> from sherpa.astro.plot import ModelPHAHistogram, ModelHistogram
+   >>> nobgplot.plot(xlog=True, ylog=True, label='Simulated without background')
+   >>> m1plot = ModelPHAHistogram()
+   >>> m1plot.prepare(data, full_model)
+   >>> m1plot.overplot(alpha=0.5, label="Grouped model")
+   >>> m2plot = ModelHistogram()
+   >>> m2plot.prepare(data, full_model)
+   >>> m2plot.overplot(alpha=0.5, label="Ungrouped model")
 
 Sometimes, more than one response is needed, e.g. in Chandra LETG/HRC-S
 different orders of the grating overlap on the detector, so they all
 contribute to the observed source
-spectrum. :py:func:`~sherpa.astro.fake.fake_pha` works if one or more responses
-are set for the input :py:class:`~sherpa.astro.data.DataPHA` object.
-
-It is also possible that the input model already includes the
-appropriate responses and scalings. In this case, ``is_source=False``
-can be set to indicate to sherpa that the model is not a naked source
-model, but includes all required instrumental effects. In this
-way, :py:func:`~sherpa.astro.fake.fake_pha` can be used with arbitrarily
+spectrum.
+The models that is passed into :py:func:`~sherpa.astro.fake.fake_pha`
+can be arbitrarily complex and can contain multiple components. For example,
+:py:func:`~sherpa.astro.fake.fake_pha` can be used
```

Comment:
grammer- replace "can be used complex models".  Perhaps "can be used with complex models"  or "can use complex models"?